### PR TITLE
Update Decision doc contents and images

### PR DESCRIPTION
**Pull request by ying-nr**
## Give us some context

* What problems does this PR solve? 

Update the contexts to be more accurate, structure to be cleaner, add images for better user reading

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 1]
```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
...
+With applied intelligence's correlation logic, alerts and events from any source are automatically grouped into related issues.  As events come into your system they are instantly correlated based on time, alert context, and relationship data. If multiple events are related then our correlation logic will funnel the related incidents into a single, comprehensive [issue](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed). This ensures that your team is alerted to relevant issues as soon as possible while also reducing distracting and redundant alerts.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:34]: there are extra spaces between some sentences in this doc, something to look for.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 2]
```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
...
+
+We call this correlation logic **decisions**. We have built-in decisions but you can also create and customize your own on the decisions page **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Decisions**. The more you configure your decisions to best suit your needs,  the better New Relic can correlate your incidents, reduce noise and provide increased context for on-call teams.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:36]: the pathing needs to be introduced and not just part of the sentence. for example, "on the decisions page. To find the decisions page, go to:..."

extra space after comma

needs serial comma after 'reduce noise'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 3]
```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
...
+  alt="A screenshot that shows applied intelligence decisions UI."
+  src={nraiDecisionsPage}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Our UI shows how each decision correlates incidents.
 </figcaption>
 
-## Key concepts [#key-concepts]
+### What is correlation and how does it work? [#what-is-correlaton]
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:48]: looks like this is an h3 now, but shouldn't that be an H2 heading?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 4]
```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
...
 
-Here are some key concepts for understanding our decisions logic:
+Your most recent and active incidents are available for our correlation logic. For example, let's say your system has received two alerts saying a synthetic monitor is failing in Australia and London. These two alerts will have created their own, unique issues. Those issues will generate their own, unique incidents based on your teams existing [incident creation policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents/#preference-target). New Relic's correlation logic will then test those incidents against each other to find similarities. In this case, it's the same monitor that is failing across multiple locations, so New Relic will merge both incidents into a single incident that contains each relevant event.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:50]: erase comma between 'own, unique', that's done twice.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 5]
```diff
@@ -21,46 +31,66 @@ import topology2 from 'images/topology-2.png'
...
+When we correlate events, we check every pair of combinations against each other and combine as many as possible. For example:
+* Our algorithm correlates incident A and B (call it "AB").
+* Our algorithm correlates incident B and C (call it "BC").
+* Because B is present in both issues, the algorithm then correlates all three incidents together into one issue.
 
-For the [incident events](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence#issue-feed) being sent into incident intelligence from your various alerting engines, the most recently created and active incidents are available for correlation. Correlation occurs between two events whenever the criteria in any decision are met.
+## Configure correlation policy [#configure-correlation]
+
+To enable correlation on [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts) based issues, you'll need to connect to correlation for the respective [alert policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name).
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:59]: should that be 'alert-based', with hyphen?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 6]
```diff
@@ -83,7 +113,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because the New Relic NRQL violation target entity names, and NRQL query are the same respectively. Does not apply to REST source. Relevant incidents from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. It helps relate incidents like same transaction query latency deviation, etc.
+        Correlation activated because the New Relic NRQL violation target entity names, and NRQL query are the same respectively. Does not apply to REST source. Relevant events from the same [NRQL alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions) will be identified. It helps relate issues like same transaction query latency deviation, etc.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:116]: we don't like to use 'etc', so we could maybe do "relate issues (for example, issues with the same transaction query latency deviation)"

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 7]
```diff
@@ -93,8 +123,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because the New Relic non-NRQL violation target entity names are the same. Does not apply to REST source. Non-NRQL entity refers to [entity](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/), typically APPLICATION, HOST types, see [New Relic GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions). Relevant incidents from the same entity will be identified. For example, host high memory incidents and host not-reporting incidents could be highly possible due to the same cause.
-      </td>
+        Correlation activated because the New Relic non-NRQL violation target entity names are the same. Does not apply to REST source. Non-NRQL entity refers to [entity](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/), typically APPLICATION, HOST types, see [New Relic GitHub repo on entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions). Relevant issues from the same entity will be identified. For example, host high memory issue and host not-reporting issue could be highly possible due to the same cause.</td>
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:126]: this paragraph needs some review. second sentence is hard to parse and grammatically incorrect. last sentence is awkward (highly possible due to)

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 8]
```diff
@@ -143,7 +172,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because the kubernetes deployments are the same. Many incidents are from single deployment changes. This decision is to reduce incidents from the same troublesome kubernetes entity deployment.
+        Correlation activated because the kubernetes deployments are the same. Many incidents are from single deployment changes. This decision is to reduce issues from the same troublesome kubernetes entity deployment.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:175]: capitalize 'Kubernetes'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 1 above)
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:34]: as a high level thing, I recommend we link to the 'issue' concept defined in https://docs.newrelic.com/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/ just to have that link if people wonder what an issue is.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 9]
```diff
@@ -123,7 +152,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
+        Correlation activated because the New Relic [condition IDs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/violation-event-attributes/#attributes) and deep link url are the same respectively. Deep link url provides timeseries and time range information in addition to [alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/). Correlating these issues make it easier for you to look at related incidents in the incident response flow with time-scoped metrics, and perform deep analysis. Deep link url can be automatically generated if incidents are triggered by New Relic alert conditions, while for REST source [deepLinkUrl](/docs/data-apis/ingest-apis/event-api/incident-event-rest-api/#api-specs) should be user defined.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:155]: capitalize 'URL'. we also seem to be using 'deep link URL' awkwardly (e.g., "and deep link URL"): should we perhaps saying "the deep link URL"? I don't know what deep link URL is.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 10]
```diff
@@ -153,7 +182,7 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
       </td>
 
       <td>
-        Correlation activated because custom application name, policy and custom id are the same respectively. This is to reduce application incidents, particularly cater to custom tag users. Learn more about [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/). Customed tag id could be defined by condition family id or other id values used as a key to identify connections between data.
+        Correlation activated because custom application name, policy and custom id are the same respectively. This is to reduce application issues, particularly cater to custom tag users. Learn more about [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/). Customed tag id could be defined by condition family id or other id values used as a key to identify connections between data.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:185]: maybe 'correlation was activated'

serial comma after 'policy'

capitalize 'ID'

grammar issue with 'particularly cater...'

For links, we usually try to make doc description more explicit, so like 'For more about tags, see [Tags].'

'customed' typo

capitalize 'ID' instances

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 11]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
+  alt="A screenshot of a suggested decision block"
+  src={screenShot20200727At65041Am}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Some example statistics from the decisions UI.
 </figcaption>
 
-Some definitions of statistics:
+To enable the suggested decision, click **Activate decision**. The decision will appear in your decision collection marked Creator as New Relic AI with a robot icon. If the decision isn't relevant to your needs, click **Dismiss**.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:247]: unclear here, 'marked Creator as New Relic AI with a robot icon'. I assume it's titled 'Creator as New Relic AI'? not sure.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 12]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
 
-* Correlation rate: The percentage of the time correlations are occurring versus not.
-* Total correlated issues: Number of issues correlated with another issue.
-* Noise reduction: Total number of issues after correlation divided by the total number of issues before correlation.
-* Correlation reason: Shows which decisions are correlating issues the most.
+## Create custom decisions [#customize]
 
-### Use suggested decisions [#suggested-decisions]
+You can reduce noise and improve correlation by building your own custom decisions. To start building a decision, go to [one.newrelic.com](https://one.newrelic.com) &gt; **Alerts & AI** &gt; **Correlate** &gt; **Decisions**, and click **Add a decision**.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:251]: does this need the 'one.newrelic.com/all-capabilities' update?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 13]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
+
+*Logic filter: Logic condition defined with an [operator](#operators) on an [attribute](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/violation-event-attributes/#attributes).
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:253]: usually for a bulletpoint list, we want some sort of intro to them. it's not clear to me what this bulletpoint list is or why it's important.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 14]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
+*Segment: A group of incidents satisfy a combination of logic filters.
 
-Information about the types of suggested decisions:
+<Callout variant="tip">
+  When building a decision, steps 1, 2, and 3 are optional on their own, but at least one must be defined in order to create a decision.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:257]: maybe reference that these steps are referring to steps coming up below?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 15]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
-  <tbody>
-    <tr>
-      <td>
-        Step 1: Filter your data (optional)
-      </td>
+For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box](#topology-requirements).
 
-      <td>
-        In this step you'll define your filters. Remember correlation occurs between two incidents. If no filters are defined, all incoming incidents will be considered by the decision.
+In addition, you can setup topology data via our NerdGraph `aiTopologyCollector`. (Search for `aiTopology` in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. Learn more about [setting up topology correlation](#topology).
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:297]: we have a doc for this https://docs.newrelic.com/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial/ that we should reference here instead of pointing them to the nerdgraph explorer.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 16]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
+### Step 5: Use advanced settings [#advanced-settings]
 
-      <td>
-        For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box](#topology-requirements).
+Use the advanced settings area to further customize how your decision behaves when correlating events. Each setting has a default value so customization is optional.
 
-        In addition, you can setup topology data via our NerdGraph `aiTopologyCollector`. (Search for `aiTopology` in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. Learn more about [setting up topology correlation](#topology).
-      </td>
-    </tr>
+* **Time window**: Sets the maximum time between two incidents created time for them to be eligible for correlation.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:323]: not able to parse the 'time window' description. unclear.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 17]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
-      </td>
-    </tr>
+## Logic operators [#operators]
 
-    <tr>
-      <td>
-        Step 5: Use advanced settings (optional)
-      </td>
+Decision provides a set of operators to help you flexibly define how an incident's attribute value evaluates in a logic filter.
+The basic ones are **equals**, **contains**, **starts with**, **ends with**, **exists**, and their negate operators accordingly. For example, **does not equal**.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:337]: should 'negate' be 'negative'?

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 18]
```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
...
 
-      <td>
-        Optional. Use the advanced settings area to further customize how your decision behaves when correlating events. Each setting has a default value so customization is optional.
+There is a similarity operator **is similar to**, the underlying [similarity algorithm](#algorithms) can be specified for this operator. By default, it uses Levenshtein Distance.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:339]: this sentence needs some work; it's two sentences run together seems like.  

what is 'levenshtein distance'? I assume it's some sort of algorithm, but might want to specify that's what it is.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 19]
```diff
@@ -957,51 +964,62 @@ Here are technical details on the similarity algorithms we use:
...
 
-* The number of captured groups must be equal for both incident attributes.
-* Each group must be equal to the corresponding group between attribute values: the value of the first captured group in the incident 1 attribute value is equal to the value of the first captured group in the incident 2 attribute.
+  <Collapser
+    id="regex-step-2"
+    title="Regex in Step 2"
+  >
+    In order for your regex to test as true, the entire attribute values for incident 1 and incident 2 must be included in the match. Also, each captured group (expressions in `( )` parentheses) must exist in both values (incident 1 and incident 2 attributes), and have the same value:
 
-For instance, if attribute value 1 is `abc-123-xyz` and attribute value 2 is `abc-777-xyz`, then `(\w+)-(?:\w+)-(\w+)` would meet the criteria:
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:986]: i haven't checked it in build yet, but you might want to review how these look in the build and see if the indents look okay.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx [hunk 20]
```diff
@@ -1011,29 +1029,30 @@ You can use the correlation assistant to more quickly analyze [incidents](/docs/
...
+    id="frequency-tips"
+    title="Attribute analysis"
+  >
+    Two types of attribute analysis appear in the UI:
 
-* **Low frequency:** As an example, an attribute with a 0% in the frequency column is likely a unique identifier or an attribute that only recently reported in your data in the last month. Choosing low frequency attributes may correlate few events.
-* **High frequency:** On the other end, an attribute with 100% frequency would be one that is present on all your data. Choosing these attributes would correlate all of your events together.
+    * **Common attributes:** This analysis simply highlights attributes and values that are the exact same between all selected incidents.
+    * **Similar attributes:** Similarity analysis uses the Levenshtein algorithm with a distance of 3 to find attributes whose values would be the same if 3 or fewer character changes are performed.
+      * Numerical values as well as single character values are filtered out of the results.
```
Comment (zuluecho9) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:1041]: these last two bulletpoints seem wrongly indented, unless they're supposed to be sub-bullets of 'similar attributes'

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:36]: done! thanks

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:48]: yes, you're right. thanks!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx]: done, thanks!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 5 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:59]: yes, you're right.

**Comment by zuluecho9** [comment]
@ying-nr Ying, I left some comments, but those are for docs team, not for you, just making sure you know that.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 8 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:175]: done.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 10 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:185]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 12 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:251]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 11 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:247]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 14 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:257]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:297]: thank you!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 18 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:339]: thanks.

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 17 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx:337]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx]: done

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx (hunk 9 above)
Comment (homelessbirds) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx]: I checked and it's being used correctly!

**Comment by clarkmcadoo** [comment]
For this, we'll need to leave the image `NRAI_Decisions_Index.png` in the project, as it still exists in the i18n versions of the edited file.  We're going to run a script to delete all unused images once the symphony work is over.  Thank you!

**Review by homelessbirds — APPROVED** [review]
